OrbitCore/Path.cpp

```diff
@@ -65,9 +65,7 @@ std::string Path::GetExecutablePath() {
 }
 
 bool Path::FileExists(const std::string& file) {
```

Comment:
I think we should add a `[[deprecated("Please use std::filesystem:.exists")]]` here. (Obviously this needs to go into the header file.

Reply:
I think it is better to just replace calls to this function with filesystem:exists(), 

./OrbitCore/SymbolHelper.cpp:23:  if (!Path::FileExists(file_name)) {
./OrbitCore/SymbolHelper.cpp:80:    if (!Path::FileExists(symbols_file_path)) continue;
./OrbitCore/OrbitModule.cpp:31:  if (!Path::FileExists(file_name)) {
./OrbitCore/LinuxUtils.cpp:121:    if (!Path::FileExists(module_path)) continue;
./OrbitCore/Path.h:45:  static bool FileExists(const std::string& a_File);
./OrbitCore/Path.cpp:67:bool Path::FileExists(const std::string& file) {
./OrbitCore/PathTest.cpp:12:  EXPECT_FALSE(Path::FileExists(filename));
./OrbitCore/PathTest.cpp:22:  EXPECT_TRUE(Path::FileExists(filename));
./OrbitCore/PathTest.cpp:28:  EXPECT_TRUE(Path::FileExists(filename));
./OrbitGl/App.cpp:252:  if (!Path::FileExists(fileName)) {

Reply:
I won't argue with that. :slightly_smiling_face:

Reply:
done